From b1f33116511937463a2fed461c9c1fe4d5189b77 Mon Sep 17 00:00:00 2001
From: Leon Schuermann <leon@is.currently.online>
Date: Sun, 7 Aug 2022 21:07:22 +0200
Subject: [PATCH] Disable LiteX CPU tests for select CPUs

These CPUs are either not packaged as part of nix-litex yet, or cannot
be tested due to requiring a non-cached toolchain or internet access
during build.
---
 test/test_cpu.py | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/test/test_cpu.py b/test/test_cpu.py
index 22dac251..b1336c9c 100644
--- a/test/test_cpu.py
+++ b/test/test_cpu.py
@@ -37,16 +37,16 @@ class TestCPU(unittest.TestCase):
 
     def test_cpu(self):
         tested_cpus = [
-            "cv32e40p",     # (riscv   / softcore)
-            "femtorv",      # (riscv   / softcore)
-            "firev",        # (riscv   / softcore)
-            "marocchino",   # (or1k    / softcore)
-            "naxriscv",     # (riscv   / softcore)
+            # "cv32e40p",     # (riscv   / softcore)
+            # "femtorv",      # (riscv   / softcore)
+            # "firev",        # (riscv   / softcore)
+            # "marocchino",   # (or1k    / softcore)
+            # "naxriscv",     # (riscv   / softcore)
             "serv",         # (riscv   / softcore)
             "vexriscv",     # (riscv   / softcore)
-            "vexriscv_smp", # (riscv   / softcore)
-            "microwatt",    # (ppc64   / softcore)
-            "neorv32",      # (riscv   / softcore)
+            # "vexriscv_smp", # (riscv   / softcore)
+            # "microwatt",    # (ppc64   / softcore)
+            # "neorv32",      # (riscv   / softcore)
         ]
         untested_cpus = [
             "blackparrot",  # (riscv   / softcore) -> Broken install?

base-commit: 08ff00317851345c7486aa58d8d2f5d07c9c8522
-- 
2.42.0

